app/contracts/work_packages/base_contract.rb

```diff
@@ -60,6 +60,10 @@ class BaseContract < ::ModelContract
     attribute :derived_estimated_hours,
               writable: false
 
+    attribute :remaining_hours
+    attribute :derived_remaining_hours,
+              writable: false
```

Comment:
Is this declaration actually necessary? The most basic use case for adding an attribute to the contract is to make it writable. But here the purpose is the exact opposite.

Reply:
I did not really think about it. I just moved it.
The lines are identical above for `:derived_estimated_hours`. 

At least it is explicit so that people in the future does not ask themself: "`:derived_remaining_hours` is not writable because it is not declared as an attribute, but is it on purpose or has it been forgotten?". Having it explicitly marked as `writable: false` removes any ambiguity about it.